src/Valit/ValitRulePropertyExtensions.cs

```diff
@@ -44,14 +44,27 @@ public static class ValitRulePropertyExtensions
             return rule;
         }
 
+        public static IValitRule<TObject, TProperty> WithMessage<TObject, TProperty>(
```

Comment:
Are you planning to add these different overloads we wrote about?

Reply:
I will do that in a later issue :-)